apriltag_ros/src/continuous_detector.cpp

```diff
@@ -67,6 +67,13 @@ void ContinuousDetector::onInit ()
   refresh_params_service_ =
       pnh.advertiseService("refresh_tag_params", 
                           &ContinuousDetector::refreshParamsCallback, this);
+  
+  target_pose_sub_ = nh.subscribe("/target/pose_base_frame", 1, &ContinuousDetector::targetPoseCallback, this);
+  
+  pnh.param<double>("fov_size_scaler", fov_size_scaler_, fov_size_scaler_);
+  // note that these are initially set incorrectly and must be squared
```

Comment:
Realized I missed this stale comment but I fixed it in a follow up MR that I'll either make today or tomorrow (branched off of this one), so this one can be ignored.